### DISCUSSION TITLE
Lunching Fluent With “Fluent Root Path” and “Computer Cluster Node Name”

### DISCUSSION THREAD

**Discussion by SebWhiteCC** [ansys/pyfluent#2332] [Q&A]
Hi,
I need to launch fluent with a specific “Fluent Root Path” and “Computer Cluster Node Name”. Is there a way to do this using PyAnsys?
Thanks.

**Comment by RobPasMue** [comment]
Transferring discussion to PyFluent's repository =)

**Comment by antoine-khairallah** [comment]
Not sure if it is  what you are looking for but here is the official documentation to start a fluent session with pyfluent python library: 
https://fluent.docs.pyansys.com/version/stable/user_guide/launching_ansys_fluent.html#solver-mode

Hope it helps
